### DISCUSSION TITLE
365 Data Science Offers Free Courses & Certificates

### DISCUSSION THREAD

**Discussion by PeterTKD** [community/community#143507] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Hey, I thought it might be useful to share that 365 Data Science is currently offering free access to all their courses until November 20. They’ve got topics like data science, analytics, ML, deep learning, and more, plus you can earn certificates.
I already binge-watched two courses and got the certificates. 
You can find website link [here](https://bit.ly/40pFqz3)

**Comment by ebndev** [comment]
Hi @PeterTKD, thanks for sharing with the community! 

Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
